src/Connection/PostObjects.php

```diff
@@ -678,6 +678,20 @@ public static function get_connection_args( $args = [], $post_type_object = null
 					'description' => __( 'Array of tag slugs, used to exclude objects in specified tags', 'wp-graphql' ),
 				];
 			}
+		} else {
+			/**
+			 * Handle cases when the connection is for many post types
+			 */
+
+			/**
+			 * Content Type $args
+			 *
+			 * @see   : https://developer.wordpress.org/reference/classes/wp_query/#post-type-parameters
+			 */
+			$args['contentTypes'] = [
```

Comment:
I believe we should be able to remove the args from here as well: https://github.com/wp-graphql/wp-graphql/blob/c8f855834f8459d40664b123487f5d9f325a8d0c/src/Connection/PostObjects.php#L94-L101